spot-client/src/common/remote-control/remoteControlClient.js

```diff
@@ -501,6 +549,13 @@ export class RemoteControlClient extends BaseRemoteControlService {
             this._waitForSpotTvTimeout = null;
         }
 
+        if (!this._p2pSignaling) {
+            this._createP2PSignalingConnection(false);
```

Comment:
Why create the connection if signaling is not enabled?

Reply:
Because then I have to do the flag logic in two places, but maybe it's better to just not create it. I will change that, because the current behavior seems weird.